internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -45,6 +45,9 @@ NetworkConfiguration:
       {{- range $sg := .Network.SecurityGroups}}
       - {{$sg}}
       {{- end}}
+      {{- if .NLB}}
+      - !Ref NLBSecurityGroup
+      {{- end}}
```

Comment:
one thing that we need to be careful of slowly is that there is a limit of 5 sg for the service.

I wonder if we should instead create a `ServiceSecurityGroup` that has ingress from `NLBSecurityGroup` so that we can always add more ingresses.
This should be a two-way door for us so it's okay if you don't update now.